src/hero/index.js

```diff
@@ -13,83 +12,83 @@ registerBlockType("mojblocks/hero", {
             type: 'string'
         },
         heroTitle: {
-            type: 'string',
-            source: 'html',
-            selector: 'h2'
+            type: 'string'
```

Comment:
How come the other attributes come out?

Reply:
These are only used when you're building blocks the "standard" block way where you manipulate the saved HTML into the database. Now that we are using PHP to do this, these properties have no use (with the exception of type which PHP needs).